src/utils/commands.ts

```diff
@@ -165,10 +165,16 @@ function attachOptions(
 }
 
 export async function getUsernameParam(interaction: CommandInteraction) {
+  const discordTagRegex = /<@!?(\d+)>/;
```

Comment:
I think it would be best to move this as a constant to the top of the file `DISCORD_TAG_REGEX`

Reply:
done